redirect.json

```diff
@@ -248,14 +262,14 @@
   {
     "redirect": "config_zwave_js",
     "name": "Z-Wave JS Configuration",
-    "badge": "Z-Wave JS Configuration",
+    "badge": "Z-Wave JS",
     "description": "show your Z-Wave JS configuration panel",
     "introduced": "2021.3"
   },
   {
     "redirect": "config_zha",
     "name": "ZHA Configuration",
-    "badge": "ZHA Configuration",
+    "badge": "ZHA",
```

Comment:
Was thinking this should probably be Zigbee?

Reply:
```suggestion
    "badge": "Zigbee",
```